provider/datagen/data/segmenter/sentence.toml

```diff
@@ -506,6 +506,15 @@ left = [
 right = [ "SContinue", "ATerm", "STerm" ]
 break_state = false
 
+[[rules]]
+# SB10
+left = [
+  "Lower_ATerm",
+  "Upper_ATerm"
+]
+right = [ "Sep", "CR", "LF" ]
```

Comment:
this doesn't seem correct, SB10 is `SATerm Close* Sp* ×  (Sp | ParaSep)`, shouldn't it be `STerm_Close_Sp` and `ATerm_Close_Sp` in `left`, and `Sep, CR, LF, Sp` on the right?

Also, what about SB9?


(In general this file is somewhat hard to read: it would be nice if we had comments on all of the tables above explaining the various states, and comments on the rules too.

Reply:
Sure. Original issue was that rules doesn't consider before Lower property. Instead of adding hacked rules, I should add additional alias rules.